### PR TITLE
misc: Add GPU info to release notes

### PR DESCRIPTION
**Pull request by powerjg**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -160,6 +160,22 @@ In this way it will be possible to connect external RNF models to the ruby inter
...
+There is a new `ViperBoard` in `gem5.prebuilt.viper`.
+This board is an initial implementation and will be improved in the next versions of gem5.
+There is an example script in `configs/example/gem5_library/x86-mi300x-gpu.py` that shows how to use the `ViperBoard`.
+See #1636.
+
+### Other GPU changes
+
+* Vega10 has been deprecated #1619
+* Replacement policy has been improved #1564
+* Improvements to the TLB #1431
```
Comment (abmerop) [inline RELEASE-NOTES.md]: I think @mattsinc is planning to revert this with #1769 .

**Review by abmerop — COMMENTED** [review]
Looks like that covers everything that's not a bug fix PR, thanks!

File: RELEASE-NOTES.md (hunk 1 above)
Comment (BobbyRBruce) [reply RELEASE-NOTES.md]: I remove this line based on this comment. Probably best not advertise things that will get reverted.

Comment (mattsinc) [reply RELEASE-NOTES.md]: Thanks @abmerop , I just merged #1769 in after manually making the requested changes.

**Review by mattsinc — APPROVED** [review]
This seems like the major things from my end after my additions!

**Pull request merged**